app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  REGEX_PATTERN = /^[\\w!#$%&'*+/=?`{|}~^-]+(?:\\.[\\w!#$%&'*+/=?`{|}~^-]+)*@(?:[a-zA-Z0-9-]+\\.)+[a-zA-Z]{2,6}$/
+  def is_email_valid? email
```

Comment:
@gabrielgaraujo is this what you meant by email validation?

Reply:
I meant [Active Record validations](https://guides.rubyonrails.org/active_record_validations.html)

Maybe you can use the [`format`](https://guides.rubyonrails.org/active_record_validations.html#format) helper

We can start with `URI::MailTo::EMAIL_REGEXP` and create our custom regex if we need it down the line.